euphonic/CITATION.cff

```diff
@@ -0,0 +1,71 @@
+cff-version: 1.2.0
```

Comment:
This file (and the LICENSE) seem to have been mistakenly copied from the project directory into the Python module folder?

Reply:
Yes